src/registration/tests/test_admin.py

```diff
@@ -409,3 +409,19 @@ def test_force_activate_users_action(self):
             None, RegistrationProfile.objects.all())
 
         self.assertEqual(RegistrationProfile.objects.count(), 0)
+
+    def test_get_inline_instances(self):
+        from django.conf import settings
+        admin_class = RegistrationAdmin(RegistrationProfile, admin.site)
+
+        # No supplements
+        settings.REGISTRATION_SUPPLEMENT_CLASS = None
```

Comment:
you should use `override_settings`

https://github.com/lambdalisue/django-inspectional-registration/blob/a5300c288c550750661e30feaefc5949cdaad6f3/src/registration/tests/test_views.py#L21